# Hypothesis Testing

# 1.State Hypotheses Statement:



#### NUll Hypotheses : The Mean weekly operating cost is $4000 ( as per theoretical model)

#### Alternative Hypotheses: The mean weekly operationg cost is greater than $4000

# 2. Calculate the Test Statistic:




### Sample  Mean = $3050

### Theoretical Mean = $4000

### Standard Deviation = 5*25= 125
### Sample size = 25


In [5]:
import numpy as np 
t=(3050-4000)/(125/(np.sqrt(25)))
t

-38.0

In [11]:
from scipy import stats
stats.norm.ppf(0.05) # Standard normal (Z) distribution table

-1.6448536269514729

# 4. Make a Decision

### Z = -38 is mush samller than -1.645
### Since Z < critical value, we reject the null hypothesis
### We accpted the Alternative hypothesis 